src/transformState.js

```diff
@@ -6,6 +6,35 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (let n = 0; n < actions.length; n++) {
+    switch (actions[n].type) {
+      case 'addProperties':
+        const actKeys = Object.keys(actions[n].extraData);
+        const actVals = Object.values(actions[n].extraData);
+
+        for (let i = 0; i < actKeys.length; i++) {
+          state[actKeys[i]] = actVals[i];
```

Comment:
use `Object.entries` inside of for...of loop isntead

Reply:
I didn't know it could be used that way, I will look into that